docs/recording/validating.md

```diff
@@ -36,6 +36,15 @@ ros2 caret check_ctf -d <path-to-trace-data>
 - Solution
   - Set `LD_PRELOAD` before running the application (See [recording section](./recording.md))
 
+### `Trace data from a package built without caret-rclcpp was detected.`
```

Comment:
I think we should describe "things that cannot be supported" or "error events" here.
Therefore, shouldn't we write "was not detected" here ...?